automation/jenkins-build.sh

```diff
@@ -2,8 +2,42 @@
 
 QEMU_VERSION='2.5.0-resin-rc1'
 QEMU_SHA256='8db1c7525848072974580b2e1c79797fc995fd299ee2e4214631574023589782'
+SUITES='sid wheezy jessie'
```

Comment:
This file should contain only bits required by jenkins to build the project. In other words, if I wanted to build this project manually I shouldn't have to need `jenkins-build.sh`. This means all these should move somewhere else except the bits that have to do specifically with how we run the jobs in jenkins